### DISCUSSION TITLE
npm install not working

### DISCUSSION THREAD

**Discussion by omdevops99** [learndevopsonline/learndevopsonline#1557] [Q&A] [answered]
Hi,
while I'm trying to npm install to install the dependency packages of Nodejs. I'm getting error as "command not found".
Public IP:  52.23.154.245
below is the snippet for your info
![Error](https://github.com/learndevopsonline/learndevopsonline/assets/141154551/8b525b5a-cd10-4f99-b375-f86d6b581e85)

**Comment by raghavarohith** [comment]


**Reply by sreedharm07** [reply, marked as answer]
@omswetha 
first  install node js  (yum install nodejs) 
then do npm install
